x-pack/performance/journeys/infra_hosts_view.ts

```diff
@@ -43,17 +43,33 @@ export const journey = new Journey({
       })
     );
   },
-}).step('Navigate to Hosts view and load 500 hosts', async ({ page, kbnUrl, kibanaPage }) => {
-  await page.goto(
-    kbnUrl.get(
-      `app/metrics/hosts?_a=(dateRange:(from:now-15m,to:now),filters:!(),limit:500,panelFilters:!(),query:(language:kuery,query:''))`
-    )
-  );
-  // wait for table to be loaded
-  await page.waitForSelector(subj('hostsView-table-loaded'));
-  // wait for metric charts to be loaded
-  await kibanaPage.waitForCharts({ count: 5, timeout: 60000 });
-});
+})
+  .step('Navigate to Hosts view and load 500 hosts', async ({ page, kbnUrl, kibanaPage }) => {
+    await page.goto(
+      kbnUrl.get(
+        `app/metrics/hosts?_a=(dateRange:(from:now-15m,to:now),filters:!(),limit:500,panelFilters:!(),query:(language:kuery,query:''))`
+      )
+    );
+    // wait for table to be loaded
+    await page.waitForSelector(subj('hostsView-table-loaded'));
+    // wait for metric charts to be loaded
+    await kibanaPage.waitForCharts({ count: 5, timeout: 60000 });
+  })
+  .step('Go to single host asset details view', async ({ page, kbnUrl, kibanaPage }) => {
+    await page.goto(
+      kbnUrl.get(
+        `app/metrics/hosts?_a=(dateRange:(from:now-15m,to:now),filters:!(),limit:500,panelFilters:!(),query:(language:kuery,query:''))`
+      )
+    );
+
+    // wait for table to be loaded
+    await page.waitForSelector(subj('hostsView-table-loaded'));
```

Comment:
Is there any reason to repeat url loading and wait for the table to be loaded again?
Based on step name, I think it is about clicking the link and waiting for asset details to be loaded.

Reply:
Thanks @dmlemeshko, that's correct I don't need to repeat the steps. I think I tested the step alone first and forgot to remove them after. I removed them now 👍